crowd_sim/envs/crowd_sim.py

```diff
@@ -649,6 +678,11 @@ def step(self, action, update=True):
             done = False
             info = Danger(dmin)
         else:
+            # delta_d = (-(norm(end_position - np.array(self.robot.get_goal_position()))) + (norm(np.array([robot_x, robot_y]) - np.array(self.robot.get_goal_position()))))
```

Comment:
should we remove this penalty statement all together? or allow it through a config file?

Reply:
I'll just remove it. According to my experiment, this penalty is not helpful.